app/styles/layout/_footer.scss

```
@@ -0,0 +1,149 @@
+.footer {
+    background: $bg-black;
+    .footer-top {
+        border-bottom: 1px solid $border-black-fading;
+        @include responsive(tablet) {
+            padding: 30px 0;
+        }
+        .footer-top__title {
+            font-weight: 700;
+            text-transform: uppercase;
+            padding-right: 35px;
+            margin: 30px 0 10px 0;
+            @include text($font-size-l, $text-main-fading);
+            @include responsive(mobile) {
+                @include text($font-size-l, $main-color);
+                padding-right: 0;
+            }
+            @include responsive(tablet) {
+                padding-right: 0;
+                margin: 0px;
+            }
+        }
+        i {
+            @include text($font-size-xl, $text-main-fading);
+            padding-right: 25px;
+            @include responsive(mobile) {
+                @include text($font-size-4xl, $text-white);
+            }
+        }
+        .footer-top__form {
+            margin-bottom: 30px;
+            @include responsive(tablet) {
+                margin: 0px;
+            }
+            input {
+                background: white;
+                width: 80%;
+                height: 45px;
+            }
+            .icon {
+                position: absolute;
+                width: 20%;
+                height: 45px;
+                line-height: 45px;
+                text-align: center;
+                background: $bg-main-hover-item;
+                i {
+                    padding: 0;
+                    color: $bg-white;
+                }
+            }
+        }
+    }
+    .footer-mid {
```

```suggestion
    &-mid {
```
Đối với những component có tên duy nhất/độc lập không bị trùng, ví dụ như ở đây là footer
Thì không cần phải lồng vào nhau như này, thay vào đó sử dụng dấu & để tránh trường hợp code bị lồng ghép nhau quá nhiều, 
Kết quả sẽ trả về khác nhau
1 cái sẽ là `.footer .footer-mid` và khi sử dụng dấu & là `.footer-mid`